src/transformer_deploy/t5_utils/t5_query_gen.py

```diff
@@ -0,0 +1,50 @@
+import json
```

Comment:
it seems this code is some kind of example code, should not be in the source code, probably more in demo folder

Reply:
moved to demo